tests/test_environments/test_gym_integration.py

```
@@ -489,6 +623,29 @@ def test_protocol_kwarg(self, _mock_port):
         env = _make_gym("localhost:8080", protocol="native")
         assert env.protocol == "native"
 
+    def test_label_query_param_host_port(self, _mock_port):
```

Ruff (0.15.21)

[warning] Fixture `_mock_port` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)